### PR TITLE
allow a host specifier on the cmd line

### PR DESCRIPTION
**Pull request by MHBauer**
move the k8s specific stuff into k8s location

assumes k8s at the local proxy 8001.

### CODE REVIEW

**Comment by MHBauer** [comment]
Can someone test this?

**Comment by duglin** [comment]
change the imports from cncf to servicebroker and it should compile

**Comment by MHBauer** [comment]
@duglin @vaikas-google @bmelville Any ideas? Merge it anyway?

**Comment by vaikas-google** [comment]
# _/home/travis/build/servicebroker/servicebroker/k8s/service_controller
./service_controller.go:34: too many arguments in call to k8s.CreateServiceStorage

This is what Travis is complaining about? 

Or is there some other issue that you're running into?

**Comment by MHBauer** [comment]
Yes. I might be having a case of tunnel vision, but that code should work.

**Comment by duglin** [comment]
ok we gotta fix this travis stuff, I'm convinced that its using code from master during imports and not from the branch.  Let me play around....

**Comment by MHBauer** [comment]
my guess, don't do 'go get' at the top level.

**Comment by MHBauer** [comment]
Why does that work dug? Doesn't make any sense.

**Comment by duglin** [comment]
There actually may have been an easier way to do it for the official SB builds but it would never work for forked builds.  In the case of forked builds the src was cloned into a path with our username in it instead of "servicebroker" which is why the "go get ./..." would always grab the master files and never see our local/changed files.  The `ln` forces our current clone to the right path with servicebroker as the dir name.

**Comment by duglin** [comment]
Ideally travis should checkout master and then merge in our branch so the paths are ok.

File: k8s/service_controller/server/k8s/storage.go [hunk 1]
```diff
@@ -11,20 +11,72 @@ import (
...
+	kind       string  `json:"kind"`
+	Versions   []VName `json:"versions"`
+}
+
+const TPRapiVersion string = "extensions/v1beta1"
+const thirdPartyResourceString string = "ThirdPartyResource"
+
+var versionMap []VName = []VName{{"v1alpha1"}}
+var serviceBrokerMeta Meta = Meta{"service-broker.cncf.org"}
+var serviceBindingMeta Meta = Meta{"service-binding.cncf.org"}
```
Comment (duglin) [inline k8s/service_controller/server/k8s/storage.go]: Missing a "services" entry but I'm assuming you'll get that in a follow-on PR.

File: k8s/service_controller/service_controller.go [hunk 2]
```diff
@@ -15,65 +12,26 @@ import (
...
 	Backend    string
+	Host       string
 }
 
 var options Options
 
 func init() {
 	flag.StringVar(&options.ConfigPath, "c", ".", "use '-c' option to specify the config file path")
 	flag.StringVar(&options.Backend, "backend", "mem", "backend to use for storing info. 'mem' for in memory backend (default), 'k8s' for kubernetes based backend")
+	flag.StringVar(&options.Host, "host", "localhost:8001", "the host of the backend to connect to")
```
Comment (duglin) [inline k8s/service_controller/service_controller.go]: Can you change this back to "8080" to match how it used to be and how Kube defaults?

File: k8s/service_controller/server/k8s/storage.go (hunk 1 above)
Comment (MHBauer) [inline k8s/service_controller/server/k8s/storage.go]: har har dug.

File: k8s/service_controller/service_controller.go (hunk 2 above)
Comment (MHBauer) [inline k8s/service_controller/service_controller.go]: is it defined anywhere? `kubectl proxy` usually brings it up on 8001 for me.

**Comment by duglin** [comment]
that's odd its not 8080 for you, I don't think I set anything up to make it do that.

**Comment by duglin** [comment]
LGTM

**Pull request merged**